app/src/main/java/fr/free/nrw/commons/utils/ImageUtils.java

```diff
@@ -36,6 +38,7 @@
     public static final int IMAGE_DARK = 1;
     public static final int IMAGE_BLURRY = 1 << 1;
     public static final int IMAGE_DUPLICATE = 1 << 2;
+    public static final int IMAGE_GEOLOCATION_DIFFERENT = 1 << 3;
```

Comment:
Why is the value `1 << 3`? It can be some constant like `-4`.

Reply:
It took some time for me to understand too, it indicates we will have a bit wise operation with these constants. 1000 | 0001 indicates that we have both a dark and wrong geotagged image. Whenever we added a new case, we can shift one more bit as 1 << 4

Reply:
Hmm, actually the implementation in my mind still uses them. We can or all constants, then we can check retult say 1011. If 4.th bit is active (we can simply do "result & 1000 != 0" for 4.th bit ), means that we will append "wrong location" message. If first bit is active (result & 0001 != 0), it means will append dark image message to list of messages.

Reply:
The check should be independent of the dark image. I am not able to understand the relation between the two.